### DISCUSSION TITLE
L-7 Forge test error

### DISCUSSION THREAD

**Discussion by Darshposhiya29** [Cyfrin/foundry-full-course-cu#199] [Q&A] [answered]
while running test for the "testPriceFeedVersionIsAccurate" i am geting the following error:

error: unexpected argument '-m' found

Usage: forge test [OPTIONS]

**Comment by alymurtazamemon** [comment, marked as answer]
@Darshposhiya29 `-m` flag is deprecated now use this one `--match-test` or short `-mt`

**Reply by Darshposhiya29** [reply]
Thanks ,
--match-test worked
-mt not working

**Reply by DevSwayam** [reply]
its ```--mt ``` 
not 
 ```-mt```
### example:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/99035115/8cebff0a-689a-4620-9acb-96aa7f0542fb)
Instead of testUserCanFundInteractions provide the function name that you want to test.
Hope this Helps.

**Reply by alymurtazamemon** [reply]
@DevSwayam Thanks for correction!

**Comment by nuprince** [comment]
match-test / -mt still not working for me ...working with a macbook ...

**Reply by alymurtazamemon** [reply]
@nuprince 

`--match-test` and `--mt`

**Reply by nuprince** [reply]
aye that works
